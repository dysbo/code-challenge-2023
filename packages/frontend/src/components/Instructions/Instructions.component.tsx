import { Box, Typography } from '@mui/material'

export const Instructions = () => (
  <Box>
    <Typography component='p'>
      Enter a numeric value into the input box.
    </Typography>
    <Typography component='p'>
      Press <strong>Calculate</strong> to see it doubled, and its double squared!
    </Typography>
  </Box>
)
