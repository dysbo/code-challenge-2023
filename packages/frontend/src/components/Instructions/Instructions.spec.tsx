import { render } from '@testing-library/react'
import { Instructions } from './Instructions.component'

describe('Instructions tests', () => {
  test('should show instructions', () => {
    expect(render(<Instructions />).container).toMatchSnapshot()
  })
})
