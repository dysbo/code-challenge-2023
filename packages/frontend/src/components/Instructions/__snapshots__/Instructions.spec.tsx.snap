// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions tests should show instructions 1`] = `
<div>
  <div
    class="MuiBox-root css-0"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Enter a numeric value into the input box.
    </p>
    <p
      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Press 
      <strong>
        Calculate
      </strong>
       to see it doubled, and its double squared!
    </p>
  </div>
</div>
`;
